sphinx_automodapi/tests/cases/public_from_private/output/api/sphinx_automodapi.tests.example_module.public.Camelot.rst

```diff
@@ -0,0 +1,7 @@
+Camelot
+=======
+
+.. currentmodule:: sphinx_automodapi.tests.example_module.public
+
+.. autoclass:: Camelot
+   :show-inheritance:
```

Comment:
@nstarman we actually generate the stub files ourselves and they contain individual eg autoclass calls so the fix has to be and is in our code.

Reply:
Then all LGTM. The results files seem reasonable :)